lib/blocks.js

```diff
@@ -157,17 +157,15 @@ api.write = callbackify(async ({ledgerNode, state, consensusResult}) => {
     meta: {blockHash, consensus: true, consensusDate: now}
   };
 
-  // FIXME: the events must be checked in the block -- if any of them is
-  //   a configuration event, then further processing in the worker should
-  //   halt, allowing for a different worker code path to run after the
-  //   new configuration has been validated and accepted (or rejected)
```

Comment:
We actually implemented what this FIXME says now, right? Can we capture this comment (with whatever tweaks are necessary) in the code documentation somewhere?